### PR TITLE
Scheduler repeats

### PR DESCRIPTION
**Pull request by proteusvacuum**
http://manage.dimagi.com/default.asp?185274
Fixes issues where repeat visits in the scheduler were not getting triggered correctly.
I'm surprised I didn't catch this when I was building it... Repeats just didn't work at all.
cc: @czue

### CODE REVIEW

**Comment by czue** [comment]
looks reasonable to me but I don't have too much context so will let @snopoke review

File: corehq/apps/app_manager/xpath.py [hunk 1]
```diff
@@ -615,18 +620,31 @@ def visit_allowed(self):
...
-        """coalesce(instance(...)/schedule/visit/[next_visits][before_window][1]/@due, [max_date]"""
-        due = (self.fixture.visit().
-               select_raw(self.next_visits()).
-               select_raw(self.before_window()).
-               select_raw("1").
-               slash("@due"))
-
-        return (u"coalesce({}, {} - {})".format(due, SCHEDULE_MAX_DATE, XPath.date(self.anchor)))
+        """if(visit/[next_visits][before_window][1]/@increment = '', @increment + last_visit_date_{form_id},
+              visit/[next_visits][before_window][1]/@due = '', @due + anchor_date, MAX_DATE)
```
Comment (snopoke) [inline corehq/apps/app_manager/xpath.py]: the `=` on  both lines should be `!=`

**Comment by snopoke** [comment]
Nothing blocking so merging

File: corehq/apps/app_manager/xpath.py (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/app_manager/xpath.py]: oops: https://github.com/dimagi/commcare-hq/pull/8909

**Pull request merged**